cmd/mesh/manifest-versions.go

```diff
@@ -105,8 +104,26 @@ func getVersionCompatibleMap(versionsURI string, binVersion *goversion.Version,
 		}
 	}
 	if myVersionMap == nil {
-		l.logAndFatal("This operator version ", binVersion.String(),
-			" was not found in the global manifestVersions map.")
+		return nil, fmt.Errorf("this operator version %v was not found in the global manifestVersions map", binVersion.String())
+	}
+	return myVersionMap, nil
+}
+
+func loadCompatibleMapFile(versionsURI string, l *logger) (b []byte, err error) {
+	if _, err = url.ParseRequestURI(versionsURI); err == nil {
+		b, err = httprequest.Get(versionsURI)
+		if err == nil {
+			return
+		}
+	}
+	l.logAndPrintf("Warning. Failed to retrieve the version map from the remote url: %v, error: %v."+
+		" Now fall back to the local file system.", versionsURI, err)
```

Comment:
nit: "Warning: failed to retrieve the version map from the remote URL (%s ): %s. Falling back to internal version map."
Here and below.

Reply:
Sure, fixed.